src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -97,12 +98,28 @@ const getInstallationsWithAdmin = async (
 		const [isAdmin, numberOfRepos] = await Promise.all([checkAdmin, numberOfReposPromise]);
 		log.info("Number of repos in the org received via GraphQL: " + numberOfRepos);
 
+		let deferredInstallUrl: string | undefined;
+
+		// TODO: we should register the request in a separate POST endpoint. Short-cutting corners now for the spike,
+		// TODO: but must be addressed before rolling out for everyone, to avoid polluting redis
+		if (await booleanFlag(BooleanFlags.ENABLE_SUBSCRIPTION_DEFERRED_INSTALL, jiraHost) && !isAdmin) {
```

Comment:
Shall we also exclude the `connected` one as well?

Reply:
This shouldn't matter because for the final feature the admin will need to explicitly create the request. We shouldn't be creating it here in the first place 🤷